### DISCUSSION TITLE
Github coin

### DISCUSSION THREAD

**Discussion by qRaVii** [IBAX-io/go-ibax#874] [Q&A]
### Describe the bug

Github coin nasıl kazanılır bilgi sahibi olanlar bizi aydınlatabilir mi 

### Reproduction

İbax netwotk üzerinde kazanılan github coin nasıl kazanılır.

### System Info

```shell
İbax , github coin
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Grkmkgl** [comment]
1

**Comment by Saracegil** [comment]
👍

**Comment by Sercio33** [comment]
👍
